docs/guides/guide-vmanomaly-vmalert.md

```diff
@@ -0,0 +1,403 @@
+# VmAnomaly Quickstart
+
+### Prerequisites
+- In this tutorial it is assumed that you have VictoriaMetrics already, and you are familiar with VictoriaMetrics ecosystem:
+  -  [victoriametrics](https://docs.victoriametrics.com/Single-server-VictoriaMetrics.html)
+  -  [vmalert](https://docs.victoriametrics.com/vmalert.html)
+  -  [vmagent](https://docs.victoriametrics.com/vmagent.html)
+  
+- It is assumed that you are familiar with Grafana and Docker
```

Comment:
I think it is worth specifying the exact versions of Grafana and Docker.

Reply:
Added grafana, vmalert, vmagent and VictoriaMetrics versions. Does Docker version really matter